### ISSUE TITLE
Freqtrade scales down Stopploss percentage in Futures with or without leverage

### ISSUE THREAD

**Issue by olav584** [freqtrade/freqtrade#8063]
## Describe your environment

  * Operating system: Ubuntu 22.04.1 LTS 5.15.0-58-generic
  * Python Version: 3.10.6
  * CCXT version: 2.4.60
  * Freqtrade Version: freqtrade 2023.1.dev
  

First of all, I would like to thank the team behind Freqtrade for everything. The Bot is great and its amazing that something like this is around for free. I have read a lot of questions and was positively suprised how well the problems were solved and how nice the team and the community were. 

## My question
I'm working for a long time on a strategy based on the FixedRiskRewardLoss-Strategy from the freqtrade-strategies-repo.
I changed the code for using it for futures with leverage, added some indicators for trade execution and changed the indicator for stoploss-detection from atr to SwingHigh/SwingLow. Works pretty fine, but there is one thing, which bugs me and i could not find a solution for.

It began when I noticed, that the trades are hitting Stoploss very fast. This was because of the leverage, which lowers the Stoploss, so I multiplied the values x5.

The calculation of the stoploss in the custom_stoploss-function seems to be correct, i checked the values via Tradingview and they are they fit to the values there. 

When I now go live with the strategy, Freqtrade takes the percent value my bot calculated, but the absolute value used by the bot is much smaller. I noticed deviations from 0,4% to about 1.3% (without leverage). 

As an example I picked a trade on the pair CEL/USDT:USDT on OKX from this morning.
[trade_export_beta7.txt](https://github.com/freqtrade/freqtrade/files/10513760/trade_export_beta7.txt)

The bot calculated a stoploss value of -0.10116542570411169, which divided by 5 for the used Leverage is about 2%.
On TradingView, this was the distance between the open and the actual 30-candle SwingHigh I am using. 
Though the Absolute StopLoss Value from freqtrade though was 0.6135, which after a few candles got triggered. The percentage the bot used was about 1,44%, which would be around 7% multiplied by Leverage. So around 3%(0,60%) for fees or what? :D

(The actual value got triggered too, but at this time, the trade was already closed a few hours. In this case it was good)

Here a Screenshot from my TV:
![image (1)](https://user-images.githubusercontent.com/102228177/214960061-e017b0f9-5f41-41c4-b35c-29b224a57764.png)

At first I thought, the issue might be the fees, but a look at the exported trades made clear that this might not be the case
The fees are indeed lower than the missing %. 

I am out of ideas and are thinking about just to increase the multiplier to my calculated Stoploss %, but there might be a better solution. And also I am interested in what causes this phenomenon.
 
Maybe Freqtrade uses something like a buffer for Stoploss? I don't know and I would be very happy if someone could help. 
I tried futures without leverage with the same result.

This is my first post, so if there is something missing I will gladly provide those informations to you.

**Comment by xmatthias** [comment]
I'm not entirely sure i understand what you mean.

As the [stoploss documentation](https://www.freqtrade.io/en/latest/stoploss/#stoploss-and-leverage) explains - it's a % of risk you're willing to loose (not the move of the pair).

So your assumption of "a 10% stoploss is about 2% move" seems right - though stoploss is not applied to price movements - but to value gained / lost.

Now there's no offset applied to stoplosses.

The bot output also contains the absolute stoploss price (the value that needs to be crossed) - which is at 0.6315 - which seems to be slightly below the value that the stop triggered (so it did trigger according to the settings - as the trade is a short).

the rest of your explanation is difficult to understand, as it reads more like an adventure book than an actual issue.

**Comment by olav584** [comment]
Thanks for your answer. Yeah, when I read my post a day later, what I have written might be a bit confusing so apologies for that :D will work on that!

So a deviation of up to 1%-1,5% between the desired Stoploss-Value and the actual Stoploss-Value is quite normal? Just asking because it killed a lot of trades (still running dry-mode, so no issue).

**Comment by xmatthias** [comment]
i'd not consider it normal - but it's also not something that i've observed in the past.

You've closed this issue however, so i assume you discovered something ... ?

**Comment by olav584** [comment]
I found a "solution" I can work with by adding an offset to the swing levels (Swing High * 1.01 / Swing Low * 0.99).
This compensates the missing % in most cases.

I was just wondering because the Swing High or Low Value is correct in the dataframe, the calcluation of the distance from this level to the open rate gives same value in % as for comparison in tradingview, but the final stoploss value the bot uses is slightly smaller, like up to around 1,5%. This is still with my offset the case, but its most times slightly bigger than the original swing values so it does the job.

I interpreted your answer like that this was normal behavior, so I did close the thread. Although english is not my mother tongue, so maybe I interpreted something wrong. 

I could also have shared my "workaround" with the community, sorry for that. First post and so :D
